src/models/Topics.ts

```diff
@@ -0,0 +1,5 @@
+export interface Topics {
```

Comment:
This could be a Value Object. Then it should be a basic type:
```ts
export interface Topic {
  label: string;
}
```

And your props will require `Topic[]`.